layouts/register/types/in-person.html

```diff
@@ -1,5 +1,14 @@
 {{ $translation := ( index $.Site.Data.translations .Site.Language.Lang ) }}
 {{ $in_person_intro := replace $translation.register.in_person__intro "%link%" (partial "links/in-person-link.html" .) }}
+
+{{ $postmarkedDeadline := cond (ne .Params.mail_postmarked_deadline nil) .Params.mail_postmarked_deadline .Params.default_mail_postmarked_deadline }}
+{{ $ipDeadline := cond (ne .Params.ip_deadline nil) .Params.ip_deadline .Params.default_ip_deadline }}
+
+{{ $dates__electionday := $translation.register.dates__electionday }}
+{{ if and (eq .Params.state_name "Louisiana") (ne $translation.register.dates__electionday_la nil) }}
+{{ $dates__electionday = $translation.register.dates__electionday_la }}
+{{ end }}
```

Comment:
Once you have added the override to the state-data.json file you can recompile the content files and then you can use the same method as the deadline to simplify this logic into a single line.